subprojects/testing-jvm/src/main/java/org/gradle/api/internal/tasks/testing/junit/IgnoredTestDescriptorProvider.java

```diff
@@ -24,10 +24,10 @@
 import java.util.List;
 
 public class IgnoredTestDescriptorProvider {
-    List<Description> getAllDescriptions(Description description, String className) {
+    List<Description> getAllDescriptions(ClassLoader classLoader, String className) {
```

Comment:
We need the legacy infrastructure to extract ignored method info from ignored class in `Vintage` engine. See https://github.com/junit-team/junit5/issues/1277

Reply:
Will be resolved in 5.1.0 of vintage-engine, 5.1.0-RC1 was released yesterday.